### ISSUE TITLE
dwmapi.dll bugs out with winamp5.552 and explorer.exe(the shell not IE)

### ISSUE THREAD

**Issue by Feay** [tumagonx/XomPie#2]
dwmapi.dll bugs out with winamp5.552 and explorer.exe(the shell not IE)
trying for a week w/o  really patching any win7 programs just having this installed created strange crashes with XP windows's shell explorer.exe how:
really random mouse clicks 
doing things that should just not do anything, like dragging an icon to the taskbar instead of an app
installing software cause theme corruption until reboot
Other problems:
winamp crashes explorer when it plays an invalid file instead of skipping it or crashing it self.
other programs that would normally sent an exception crash explorer.exe.
...
after manually deleteing all the xcompie files listed n the INI i found that whilst i was playing wnamp it, winamp decided to be using dwmapi.dll! testing le crashes  with an invalid mpeg file after deleting dwmapi and with out.
***
lastly there is NO uninstaller shortcut made, but the ini mentions such that is placed in the windows dir.
would NSIS  http://nsis.sourceforge.net/Main_Page

**Comment by tumagonx** [comment]
Agree that it shouldn't use same dll filename. Winamp does try use dwmapi.dll therefor uninstallation attempt when Winamp active will not remove dwmapi.dll

But I cant reproduce:
1. Explorer issue, why would explorer check something that it's not designed for (or existed at its time)?, unless there is 3rd party shell extension bring down explorer when it stumble on dummy dwmapi.dll.
2. Uninstaller, There is no Xompie > Uninstall Xompie after installing it?

**Comment by Feay** [comment]
yeah there is not uninstall script made with Xompie and it does not
also show in remove programs, I had to look into xompie.inf to see
what files it added! then noticed that dwmapi.dll was being used by
winamp...as i deleted them by hand


Now that I think about it the issue might me that:\
winamp pulls up a video and uses FFMPEG
but if the video was invalid, instead of skipping it then it would
crash winamp and explorer would pop up with an error which would crash
it saying the same things i cite below..
so the connection i think could be ffmpeg.

in additon to that explorer would randomly anyhow even if winamp wasn't open.
 crash and tell me an error in a module but not say which it's file
version, and that version number matches dwmapi.dll  these random
crashes of explorer occur when right clicking folders  and files on
the desktop and miniuzing windows and maximizing windows, i got sounds
for those last two events.....hmm i wonder if realtek audio manager
hd....is to blame as well as winamp's interaction with dwmapi.dll, or
even my shell extensions for right click menu!

My usual programs are firefox, meshlab, blender, winamp, hex chat, and
auto start realtek audio manager hd....

I am running XPS3.


anyhow letting you know!

and a packaged installer would be nice try like the one i posted
before that way one has an uninstall route
